pkg/oc/cli/admin/policy/reconcile_sccs.go

```diff
@@ -211,8 +211,7 @@ func (o *ReconcileSCCOptions) ChangedSCCs() (
 	toUpdateSCCs := []*securityv1.SecurityContextConstraints{}
 	toCreateSCCs := []*securityv1.SecurityContextConstraints{}
 
-	groups, users := bootstrappolicy.GetBoostrapSCCAccess(o.InfraNamespace)
-	bootstrapSCCs := bootstrappolicy.GetBootstrapSecurityContextConstraints(groups, users)
+	bootstrapSCCs := bootstrappolicy.GetBootstrapSecurityContextConstraints()
```

Comment:
the `if o.Union {` logic will need to be removed

Reply:
I kept those parts that are irrelevant to this change